safe-global/safe-wallet-web
# Refactor(Sidebar): use feature directory structure for MyAccounts

## How this PR fixes it
The component was getting too big to easily manage, with all the styles in one file. This PR splits it up and uses the structure used for other new features.

Also fixes the styles for status and queue action chips on account items to match the designs.
https://www.figma.com/design/LJqeprG4WezKdxuCRoUQPq/Testing%3A-User-accounts?node-id=3628-8109&t=dg1VYSZWvKLzK7sP-1

## How to test it
- make sure everything works as before
- make sure the status and queue chips are in line with the address for larger screens, and underneath the account icon for smaller screens

## Screenshots
Design changes:
![image](https://github.com/user-attachments/assets/3cb22efe-15e1-4298-a8cc-92c805d517e5)

![image](https://github.com/user-attachments/assets/e548c2f6-024b-4809-992c-9d00ec1c9437)

## Checklist
